Changes.md

```diff
@@ -1,7 +1,10 @@
 1.4.x.x (relative to 1.4.5.0)
 =======
 
+API
+---
 
+- GafferUI : Menu items with "checkBox" enabled now draw their checked indicator in place of their "icon" when both are defined.
```

Comment:
I think `GafferUI :` is a bit vague as the context for this one -`Menu :` gives more of an at-a-glance reading.